src/SFA.DAS.ProviderCommitments/SFA.DAS.ProviderCommitments.UnitTests/Services/WhenICreateACsvFile.cs

```diff
@@ -67,7 +67,7 @@ public void SetUp()
             };
         }
 
-        [Test]
```

Comment:
Not sure this should be here?

Reply:
Tests are fixed on a different branch, we agreed to add ignore attribute to them elsewhere to avoid duplicated effort.